src/components/project-details/project-details.test.tsx

```diff
@@ -7,15 +7,31 @@ import { render, screen } from '@testing-library/react';
 import { Project } from 'utils';
 import ProjectDetails from './project-details';
 
-it('Renders title', () => {
-  const exProject: Project = {
-    wbsNum: '1.1.0',
-    name: 'Impact Attenuator',
-    projectLead: 'Person Allen',
-    projectManager: 'Person Gilbert',
-    duration: 2
-  };
-  render(<ProjectDetails project={exProject} />);
-  const titleElement = screen.getByText(/Project Details/i);
-  expect(titleElement).toBeInTheDocument();
+// Build example test data
+const exampleProject1: Project = {
+  wbsNum: '1.1.0',
+  name: 'Impact Attenuator',
+  projectLead: 'Person Allen',
+  projectManager: 'Person Gilbert',
+  duration: 2
+};
+
+describe('project details component', () => {
+  test('Renders title', () => {
```

Comment:
let's use the `it()` syntax just to keep things consistent. Behind the scenes, they're aliases so they point to the same function.